spec/decorators/application_decorator_spec.rb

```diff
@@ -35,22 +35,22 @@
   # as ApplicationDecorator is an abstract class, some tests also use an actual decorator
   describe ".decorate_collection" do
     before do
-      create(:sponsorship, level: :gold)
+      create(:article, approved: true)
     end
 
     it "receives an ActiveRecord relation and returns an array of decorated records" do
-      relation = Sponsorship.gold
+      relation = Article.approved
 
       decorated_collection = described_class.decorate_collection(relation)
-      expect(decorated_collection.map(&:class)).to eq([SponsorshipDecorator])
+      expect(decorated_collection.map(&:class)).to eq([ArticleDecorator])
```

Comment:
We changed the example to use the article decorator instead of sponsorship decorator.